release/ray_release/cluster_manager/cluster_manager.py

```diff
@@ -55,6 +55,11 @@ def set_cluster_env(self, cluster_env: Dict[str, Any]):
         self.cluster_env["env_vars"][
             "RAY_memory_monitor_interval_ms"
         ] = self.cluster_env["env_vars"].get("RAY_memory_monitor_interval_ms", "250")
+        self.cluster_env["env_vars"][
+            "RAY_memory_usage_threshold_fraction"
+        ] = self.cluster_env["env_vars"].get(
+            "RAY_memory_usage_threshold_fraction", "0.95"
```

Comment:
maybe we should remove it?

Reply:
For now we can't , product is override this to 1. going to ping @allenyin55 so we can remove this override